### ISSUE TITLE
Improve usability of mobile map

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12919]
## Description
On desktop, the map and the results list are on the same screen, visible at the same time and the location marker is associated with the location search results information. However, this is not the case on mobile - the map and the results are in separate tabs. When looking at the list of results, a user must remember which location A is, then go to the map, find A, maybe the user has forgotten some of the information - what was the name of this location again? What is the address? Having this information on 2 screen increases cognitive load for the user.

### Possible solutions
Would it be possible to make the maps icons actionable? Available to keyboard/mouse/touch and when they are activated can we bring up a card with the location information?

There was [an ticket with this idea](https://github.com/department-of-veterans-affairs/va.gov-team/issues/20254) and a possible design that was in the other repo

## Acceptance Criteria

- [ ] Decrease cognitive load for finding locations on mobile

**Comment by laflannery** [comment]
@thejordanwood

**Comment by mmiddaugh** [comment]
Related feedback from Collab cycle Design intent for Facility Locator iteration: (department-of-veterans-affairs/va.gov-team#62285) 
- Map markers should receive keyboard focus and be keyboard-interactive ([enter] key to activate marker and move focus to the corresponding entry in the results list).
- Map markers should have a unique accessible name.
- If map markers can receive keyboard focus, then provide keyboard users a skip link to jump them past the map and to the pagination (so they don't have to tab through 10 markers plus map controls if they don't want to).

**Comment by laflannery** [comment]
Closing in favor of #15304
